aufs.go

```diff
@@ -404,7 +403,7 @@ func supported() error {
 			return nil
 		}
 	}
-	return errors.Errorf("aufs is not supported" + probeError)
+	return fmt.Errorf("aufs is not supported" + probeError)
```

Comment:
I think probeError should be an error not a string?
the wrapped sprintf above isn't in the right wrap format `string: %w`  not sure what the benefit was to add out on error to the probeError string...
if it was then this line would be ```fmt.Errorf("aufs is not supported: %w", probeError)```

Reply:
Done